### PR TITLE
TE5-T422 Recyclerview Added

### PR DESCRIPTION
**Pull request by vishal2001prajapati**

### CODE REVIEW

File: app/src/main/java/com/example/myapplication3/RecyclerView/ItemAdapter.kt [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    RecyclerView.Adapter<ItemAdapter.MyviewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): MyviewHolder {
+        val viewHolder = LayoutInflater.from(parent.context).inflate(R.layout.item_recyclerview, parent, false)
+        return MyviewHolder(viewHolder)
+    }
+
+    @SuppressLint("NotifyDataSetChanged")
+    override fun onBindViewHolder(holder: MyviewHolder, position: Int) {
+        holder.itemtext.text = itemList[position].str_text
```
Comment (Purvesh-Dodiya) [inline app/src/main/java/com/example/myapplication3/RecyclerView/ItemAdapter.kt]: you can put this line in below apply block

File: app/src/main/java/com/example/myapplication3/RecyclerView/RecyclerViewActivity.kt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+import kotlinx.android.synthetic.main.activity_recycler_view.*
+
+class RecyclerView : AppCompatActivity() {
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_recycler_view)
+        val actionBar = supportActionBar
+        actionBar?.apply {
+            title = "RecyclerView"
```
Comment (Purvesh-Dodiya) [inline app/src/main/java/com/example/myapplication3/RecyclerView/RecyclerViewActivity.kt]: static string

File: app/src/main/java/com/example/myapplication3/RecyclerView/RecyclerViewActivity.kt [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+            add(RecycleModel(R.drawable.camera, "Vaibhav",false))
+            add(RecycleModel(R.drawable.bookmark, "Parth",false))
+            add(RecycleModel(R.drawable.camera, "Yash",false))
+            add(RecycleModel(R.drawable.bookmark, "Dhruval",false))
+            add(RecycleModel(R.drawable.camera, "Nitin",false))
+            add(RecycleModel(R.drawable.bookmark, "Nayan",false))
+            add(RecycleModel(R.drawable.camera, "Prathmesh",false))
+        }
+        adapter = ItemAdapter(list)
+        recycler_view.layoutManager = LinearLayoutManager(this)
```
Comment (Purvesh-Dodiya) [inline app/src/main/java/com/example/myapplication3/RecyclerView/RecyclerViewActivity.kt]: use apply

**Review by Purvesh-Dodiya — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/myapplication3/RecyclerView/ItemAdapter.kt (hunk 1 above)
Comment (vishal2001prajapati) [reply app/src/main/java/com/example/myapplication3/RecyclerView/ItemAdapter.kt]: in the holder holder itemtext and new itemcheck is diffrent statement so i give the apply is not work

Comment (Purvesh-Dodiya) [reply app/src/main/java/com/example/myapplication3/RecyclerView/ItemAdapter.kt]: Pending

**Review by Purvesh-Dodiya — CHANGES_REQUESTED** [review]

**Pull request merged**